src/test/java/io/reactivex/internal/subscribers/BoundedSubscriberTest.java

```diff
@@ -0,0 +1,292 @@
+        final List<Object> received = new ArrayList<Object>();
+
+        BoundedSubscriber<Object> o = new BoundedSubscriber<Object>(new Consumer<Object>() {
+            @Override
+            public void accept(Object o) throws Exception {
+                received.add(o);
+            }
+        }, new Consumer<Throwable>() {
+            @Override
+            public void accept(Throwable throwable) throws Exception {
+                received.add(throwable);
+            }
+        }, new Action() {
+            @Override
+            public void run() throws Exception {
+                received.add(1);
+            }
+        }, new Functions.BoundedConsumer(128) {
+            @Override
+            public void accept(Subscription subscription) throws Exception {
+                throw new TestException();
+            }
+        });
+
+        assertFalse(o.isDisposed());
+
+        Flowable.just(1).subscribe(o);
+
+        assertTrue(received.toString(), received.get(0) instanceof TestException);
+        assertEquals(received.toString(), 1, received.size());
+
+        assertTrue(o.isDisposed());
+    }
+
+    @Test
+    public void onNextThrows() {
+        final List<Object> received = new ArrayList<Object>();
+
+        BoundedSubscriber<Object> o = new BoundedSubscriber<Object>(new Consumer<Object>() {
+            @Override
+            public void accept(Object o) throws Exception {
+                throw new TestException();
+            }
+        }, new Consumer<Throwable>() {
+            @Override
+            public void accept(Throwable throwable) throws Exception {
+                received.add(throwable);
+            }
+        }, new Action() {
+            @Override
+            public void run() throws Exception {
+                received.add(1);
+            }
+        }, new Functions.BoundedConsumer(128) {
+            @Override
+            public void accept(Subscription subscription) throws Exception {
+                subscription.request(bufferSize);
+            }
+        });
+
+        assertFalse(o.isDisposed());
+
+        Flowable.just(1).subscribe(o);
+
+        assertTrue(received.toString(), received.get(0) instanceof TestException);
+        assertEquals(received.toString(), 1, received.size());
+
+        assertTrue(o.isDisposed());
+    }
+
+    @Test
+    public void onErrorThrows() {
+        List<Throwable> errors = TestHelper.trackPluginErrors();
+
+        try {
+            final List<Object> received = new ArrayList<Object>();
+
+            BoundedSubscriber<Object> o = new BoundedSubscriber<Object>(new Consumer<Object>() {
+                @Override
+                public void accept(Object o) throws Exception {
+                    received.add(o);
+                }
+            }, new Consumer<Throwable>() {
+                @Override
+                public void accept(Throwable throwable) throws Exception {
+                    throw new TestException("Inner");
```

Comment:
Place a breakpoint here and debug the test. It should reveal why the code coverage doesn't seem to execute that code in `BoundedSubscriber`.